setup3.py

```diff
@@ -62,11 +62,12 @@ def main():
             'build_scripts': PickyBuildScripts,
         },
         packages=find_packages(),
-        conditionalExtensions=getExtensions(),
         install_requires=["zope.interface >= 4.0.2"],
-        zip_safe=False,
-        include_package_data=True,
+        conditionalExtensions=getExtensions(),
         scripts=getScripts(),
+        include_package_data=True,
+        zip_safe=False,
+        extras_require=_EXTRAS_REQUIRE,
```

Comment:
I changed the order to make the order the same as in setup.py.
Since setup3.py is using more and more stuff from dist.py, I made the change to make it easier to compare setup.py and setup3.py